inference-engine/src/vpu/graph_transformer/include/vpu/model/data.hpp

```diff
@@ -82,10 +82,19 @@ struct ShapeLocation final {
     int dimsOffset;
     Location stridesLocation;
     int stridesOffset;
+
+    bool operator==(const ShapeLocation& shapeLocation) {
+        return std::tie(dimsLocation, dimsOffset, stridesLocation, stridesOffset) ==
+               std::tie(shapeLocation.dimsLocation, shapeLocation.dimsOffset, shapeLocation.stridesLocation, shapeLocation.stridesOffset);
+    }
+
+    bool operator!=(const ShapeLocation& shapeLocation) {
+        return !(*this == shapeLocation);
+    }
 };
 
-static constexpr ShapeLocation defaultShapeLocation = {
-        Location::None, 0, Location::None, 0
+static const ShapeLocation defaultShapeLocation = {
```

Comment:
Why you changed `constexpr` with `const`?

Reply:
Returned back to `constexpr`